content/rv/api/how-to/create-and-manage-cloud-accounts.md

```diff
@@ -1,68 +1,72 @@
 ---
```

Comment:
@dannycohen Please review my changes here. I'll use this as a base for other changes.